### PR TITLE
add cypress tests for no integration sites.

### LINKED ISSUE #779: Add Cypress testing to non-interface repos
**Is your feature request related to a problem? Please describe.**
Continuous integration for the non-interface repos do not use Cypress

**Describe the solution you'd like**
CI should run a Cypress suite for each app.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #779

### CODE REVIEW

File: packages/delete-your-data/cypress/integration/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-undef */
+
+describe('Visiting /', () => {
+  it('renderes /', () => {
+    cy.visit('/');
```
Comment (nisarhassan12) [inline packages/delete-your-data/cypress/integration/index.spec.ts:5]: @shicholas This seems to be failing not sure why i.e visiting `/` seems to work in the browser but not sure why cypress fails to access this? The page `/` is created from `src/content/index.mdx`. 

image

Comment (nisarhassan12) [reply packages/delete-your-data/cypress/integration/index.spec.ts:5]: The same thing is true for the `Justice for Rickie Slaughter` website.

Comment (shicholas) [reply packages/delete-your-data/cypress/integration/index.spec.ts:5]: I'm looking for a `baseUrl` in the `cypress.json` like this https://github.com/NeonLaw/codebase/blob/development/packages/interface/cypress.json#L2 have you tried that?

Comment (nisarhassan12) [reply packages/delete-your-data/cypress/integration/index.spec.ts:5]: Thanks. @shicholas It worked not having the `baseUrl` was the problem.

**Review by shicholas — APPROVED** [review]
thank you

**Pull request merged**